### DISCUSSION TITLE
How to add @defer directive support?

### DISCUSSION THREAD

**Discussion by FezVrasta** [MichalLytek/type-graphql#1675] [General]
Hi, I enabled `@defer` on my Yoga server, but the generated schema doesn't include the `@defer` directive definition. I read on the docs directives have to be manually defined but I don't understand what am I supposed to do here?

**Comment by Ricolo53** [comment]
Copy [this code](https://github.com/ardatan/graphql-tools/blob/master/packages/utils/src/directives.ts#L13C8-L29C4) and add it on `buildSchema.directives` array
